src/tools/sfdk/command.cpp

```diff
@@ -1660,12 +1660,24 @@ Worker::ExitStatus BuiltinWorker::runTools(const QStringList &arguments_, int *e
     }
 
     if (arguments.first() == "remove") {
-        if (!P::checkPositionalArgumentsCount(arguments, 2, 2))
+        QCommandLineParser parser;
+        QCommandLineOption snapshotsOfOption(QStringList{"snapshots-of"});
```

Comment:
This should work too https://doc.qt.io/qt-5/qcommandlineoption.html#QCommandLineOption

```suggestion
        QCommandLineOption snapshotsOfOption("snapshots-of");
```

Reply:
I'm using QStringList there for consistency ;) No, just kidding, will change